x-pack/test/security_solution_cypress/config.ts

```diff
@@ -47,6 +47,9 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
         `--xpack.securitySolution.enableExperimental=${JSON.stringify([
           'chartEmbeddablesEnabled',
         ])}`,
+        `--xpack.securitySolution.offeringSettings=${JSON.stringify({
+          disableTimelineSaveTour: true,
+        })}`,
```

Comment:
The goal of the `offeringSettings` flags is to allow us to enable/disable different features in serverless vs ESS "offerings". This new flag has been created only to disable a feature for cypress tests, which is not a correct use of this tool.

If it is temporary I am fine keeping this `disableTimelineSaveTour` flag here because it solves the problem, but after the tests are adapted we will have to clean it and remove the flag. 

If this was planned to be permanent I would suggest finding another way to disable the tour for cypress. Maybe we could set the localStorage key that is already being used, or just add a task to click the finish button in the tests. 

wdyt?

Reply:
@semd I am using `offeringSettings` to disable the tour to follow a previous pattern that is used in Kibana with `disableWelcomeScreen`. That one is also only used for disabling a tour in tests.

I agree though that pre-populating `localStorage` would be the better way but I didn't manage to make that work at this point. The config flag is the solution that made it work in cypress and ftr tests.

Reply:
@semd However, I agree that the implied function of the name `offeringSettings` and what the `disableWelcomeScreen` flag do, do not overlap. I could not find an easier way to get access to config variables that are passed to Kibana. Are you aware of another way to get this flag?

Reply:
`disableWelcomeScreen` flag is set in a direct config of the `home` plugin, we could set a new config in a similar way for the security_solution plugin, or we could also just use `experimentalFeatures`, which has a more generic scope. Let me pull your branch and try a couple of things.